ansible/vars/production/production_public.yml

```diff
@@ -35,7 +35,6 @@ primary_ssl_env: "production"
 dev_group: 'dimagidev'
 dev_users:
   present:
-    - ansible
```

Comment:
What do you think about putting it in `group_vars/all.yml`? Looks like [`dev_group` is already in there](https://github.com/dimagi/commcarehq-ansible/blob/30a9104cffbfc3d7a8de9858485742b57f5298ec/ansible/group_vars/all.yml#L22).